intermediate_source/DCP_tutorial.rst

```diff
@@ -0,0 +1,241 @@
+Getting Started with Distributed Checkpoint (DCP)
+=====================================================
+
+**Author**: `Iris Zhang <https://github.com/wz337>`__, `Rodrigo Kumpera <https://github.com/kumpera>`__, `Chien-Chin Huang <https://github.com/fegin>`__
+
+.. note::
+   |edit| View and edit this tutorial in `github <https://github.com/pytorch/tutorials/blob/main/intermediate_source/DCP_tutorial.rst>`__.
+
+Checkpointing AI models during distributed training could be challenging, as parameters and gradients are partitioned across GPUs and the number of GPUs available could change when you resume training.
+Pytorch Distributed Checkpointing (DCP) can help make this process easier.
+
+In this tutorial, we show how to use DCP APIs with a simple FSDP wrapped model.
+
+
+How DCP works
+--------------
+
+:func:`torch.distributed.checkpoint` enables saving and loading models from multiple ranks in parallel.
+In addition, checkpointing automatically handles fully-qualified-name (FQN) mappings across models and optimizers, enabling load-time resharding across differing cluster topologies.
+
+DCP is different from :func:`torch.save` and :func:`torch.load` in a few significant ways:
+* It produces multiple files per checkpoint, with at least one per rank.
+* It operates in place, meaning that the model should allocate its data first and DCP uses that storage instead.
```

Comment:
Those two points are quite important and it's worth repeating them once again during the tutorial.

We should add some comments in the load code about the in-place aspect and how different it is from torch.load.

For the multiple files, after saving, we should encourage the reader to look at the generated files and show a screenshot / screen dump listing all the files.